apps/publisher/modules/asset-api.js

```diff
@@ -391,6 +396,7 @@ var result;
      * @return updated-asset
      */
     api.update = function (options, req, res, session) {
+        metrics.start("asset-api","update");
```

Comment:
Thanks @lahirus , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/d7a276e7651cd0c89c6e111c33005be71251a552)